public/javascript/pump/view.js

```diff
@@ -1588,13 +1588,38 @@
         tagName: "div",
         className: "row comment-form",
         events: {
-            "submit .post-comment": "saveComment"
+            "submit .post-comment": "saveComment",
+            "click .close-btn": "cancelComment"
         },
         ready: function() {
             var view = this;
             view.$('textarea[name="content"]').wysihtml5({
                 customTemplates: Pump.wysihtml5Tmpl
             });
+
+            // Close other comments instances if are empty
```

Comment:
This kind of seems like we're doing too much? And I don't think this will happen often in practice

Reply:
I'm not sure, works fine but I don't know if is necessary

Reply:
@vxcamiloxv honestly I'd rather just err on the side of having less code

Reply:
Github/Diaspora keep open the comments boxes, so I think some users leave open it for remember comment later, What do you think?

Reply:
@vxcamiloxv yeah, that was the usecase I had in mind when I said "I wonder if we're doing too much"